### DISCUSSION TITLE
Durée de fonctionnement des robots

### DISCUSSION THREAD

**Discussion by philippejuhel** [robot-soccer-kit/robot-soccer-kit#102] [Q&A]
Bonjour,

Quelle est (approximativement) la durée de fonctionnement d'un robot (la durée des batteries)? 

Merci,

Philippe

**Comment by Gregwar** [comment]
Bonjour,

![image](https://github.com/robot-soccer-kit/robot-soccer-kit/assets/367022/16bb067d-aed2-454b-9352-3747cd7097c1)


La consommation des robots est entre 150mA et 600mA (du robot inactif au robot qui roule et tire)
Les batteries que vous avez ont une capacité théorique de 2850mAh ce qui donnerait une autonomie théorique entre 19h et 4.75h

Il faut réduire ce nombre dans la pratique, mais je dirais qu'en période de compétition (usage intense) il nous est arrivé de durer environ 3h avant de changer les batteries.

En usage "pédagogique", avec des étudiants qui ne font pas bouger les robots en permanence, je dirais qu'une demie journée est facilement atteignable.

Notez également que vous disposez de 2 jeux de batteries, ce qui permet de les charger en même temps que vous les utilisez, et qu'elles chargent plus vite que les robots ne les déchargent. 

Grégoire
